config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -0,0 +1,236 @@
+#! /bin/bash -e
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The presubmit jobs for the different Kubernetes-CSI repos are all
+# the same except for the repo name. As Prow has no way of specifying
+# the same job for multiple repos and manually copy-and-paste would be
+# tedious, this script is used instead to generate them.
+
+base="$(dirname $0)"
+
+# All kubernetes-csi repos for which Prow gets configured.
+repos="
+cluster-driver-registrar
+csi-driver-fibre-channel
```

Comment:
do you want to add all the repos at once, or do them in phases?  adding them all at once means we can't flip the always_run bit until they're all done.

Reply:
Correct. I intend to create all the necessary PRs, make sure that they pass in all repos and are merged, then flip `always_run`. That way we will have some repos where Prow could already be enabled and isn't yet, but hopefully that will be only for a short while.